data/repositories.toml

```diff
@@ -731,4 +731,5 @@ repositories = [
   'github.com/zsh-users/zsh-completions',
   'github.com/zsh-users/zsh-syntax-highlighting',
   'github.com/zulip/zulip',
+  'github.com/MakeContributions/ideahub',
```

Comment:
Can you please add the repository in the correct lexicographic order?

Reply:
Ok